qmail-lspawn.8

```diff
@@ -34,7 +38,6 @@ mechanism; if the address is not listed there, it invokes
 then runs
 .B qmail-local
 under the user's uid and gid.
-It does not set up any supplementary groups.
```

Comment:
Why do you remove this line?

Reply:
Oops. That's by mistake

Reply:
fixed it. I think I am making too many mistakes. Somehow I lost the changes made to the three man pages that I had made around few weeks earlier. I have pushed the corrections just now